tests/unit/core/utils/test_index.js

```diff
@@ -55,48 +52,11 @@ import {
 
 describe(__filename, () => {
   describe('addonHasVersionHistory', () => {
```

Comment:
Let's delete the entire `describe` block as well as the `addonHasVersionHistory` function.

Reply:
done.